addons/website_quote/models/sale_order.py

```diff
@@ -77,6 +77,11 @@ def _compute_amount_undiscounted(self):
             total += line.price_subtotal + line.price_unit * ((line.discount or 0.0) / 100.0) * line.product_uom_qty  # why is there a discount in a field named amount_undiscounted ??
         self.amount_undiscounted = total
 
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
+        super(SaleOrder, self).onchange_partner_id()
+        self.note = self.template_id.note or self.note
```

Comment:
So we want to have the Terms & Conditions reset if the customer changes, unless there is nothing in the template then we keep what's already in it.

I don't see this in the specs, is it normal ?

Also, this will not work as it is already done is sale's onchange_partner_id definition.